gossip/src/duplicate_shred.rs

```diff
@@ -595,7 +595,9 @@ pub(crate) mod tests {
                 ),
                 new_rand_data_shred(
                     &mut rng,
-                    next_shred_index + 1,
+                    // With Merkle shreds, last erasure batch is padded with
+                    // empty data shreds.
+                    next_shred_index + if merkle_variant { 30 } else { 1 },
```

Comment:
why are these 30 and not 32?

Reply:
Because when there is no padding, `new_rand_data_shred` creates 2 shreds and it always returns the last shred.
so the last shred index is equal to
```rust
next_shred_index + 32 - 1        // with padding
next_shred_index + 30 + 2 - 1  // without padding + 30
```